### PR TITLE
Fix markeplace layout

### PR DESCRIPTION
**Pull request by dherault**
## Summary

Fixes the marketplace scene issue with the new design
## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Review by maciaszczykm — APPROVED** [review]

**Comment by dherault** [comment]
Not ready yet

**Comment by dherault** [comment]
Ready

File: www/src/components/marketplace/MarketplaceRepositories.js [hunk 1]
```diff
@@ -30,6 +35,8 @@ const filterTokenStyles = {
   backgroundColor: 'fill-one-selected',
```
Comment (dogmar) [inline www/src/components/marketplace/MarketplaceRepositories.js:35]: Line 33 above this needs to be removed. It was an old hack to match the filter token heights to the text field, and is now causing them to be too tall.

Comment (dogmar) [reply www/src/components/marketplace/MarketplaceRepositories.js:35]: This line: `minHeight: '42px',`

**Review by maciaszczykm — DISMISSED** [review]

**Review by maciaszczykm — COMMENTED** [review]
Besides issue with scroll I've noticed that if you filter repos and i.e. only 2 will show as a result the whole page layout changes. This probably should be avoided.

Zrzut ekranu 2022-07-27 o 09 27 36
Zrzut ekranu 2022-07-27 o 09 27 40

Both screenshots were taken directly from packages-redesign branch.

**Comment by dherault** [comment]
Thanks for the reviews, will work on it asap

**Comment by dherault** [comment]
Ready for rereview

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**